backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -223,6 +223,11 @@ def get_ui_display(self, data):
             ui_display = json.loads(get_value(data, "metadata._ui_display", ""))
             if is_superuser_or_cataloger_logged_in():
                 ui_display["can_edit"] = True
+                search_type = request.values.get("search_type", "", type=str)
+                if search_type == "hep-author-publication":
+                    ui_display["curated_relation"] = get_value(
+                        data, "metadata.curated_relation", False
+                    )
```

Comment:
The permission check will change in the future, it will be better if we isolate it from `is_superuser_or_cataloger_logged_in` check here